pkg/storage/types.go

```diff
@@ -131,3 +132,13 @@ type BadgerDBWithCache interface {
 	CacheInstance() *cache.Cache
 	Name() string
 }
+
+type ApplicationMetadata struct {
+	// Fully Qualified Name. Eg app.cpu ({__name__}.{profile_type})
+	FQName string `gorm:"index,unique;not null;default:null" json:"name"`
+
+	SpyName         string                   `json:"spyName,omitempty"`
+	SampleRate      uint32                   `json:"sampleRate,omitempty"`
+	Units           metadata.Units           `json:"units,omitempty"`
+	AggregationType metadata.AggregationType `json:"-"`
```

Comment:
any particular reason for not including this in JSON?

Reply:
Oh, we're not using this in the API yet, so I guess this is fine..

Yeah, I'd say, if we do ever use this in the HTTP apis we should probably expose AggregationType and also we should not `omitempty` because zod will complain about it if they are empty.

Reply:
Reasoning is simple: since this is an endpoint tailored for the frontend, and since that information is not used in the frontend, there's no need to expose it.

I previously talked with @kolesnikovae and aggregationType is a storage thing, I don't think the frontend user has any use for this info. I may be mistaken.

```
❯ grep -ri 'aggregationType' webapp/ | wc -l
0
```

Reply:
yeah that's fair